src/routes/(inner)/api/use-click/+page.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+	import CodeBlock from '$docs/components/CodeBlock/CodeBlock.svelte';
+	import Table from '$docs/components/Table/Table.svelte';
+	import { tableOptions } from './data.js';
+</script>
+
+<div class="space-y-10">
+	<!-- Header -->
+	<header class="card card-gradient space-y-8">
+		<h1 class="h1"><span>useClick</span></h1>
+		<p>Opens or closes the floating element when clicking the reference element.</p>
+		<CodeBlock lang="ts" code={`import { useClick } from '@skeletonlabs/floating-ui-svelte';`} />
+	</header>
+	<!-- Usage -->
+	<section class="space-y-8">
+		<h2 class="h2">Usage</h2>
+		<CodeBlock
```

Comment:
Let's go ahead and migrate the multiple individual Code Blocks to a single instance that imports `Example.svelte?raw`. We can use this as reference to update all other examples soon. It'll also better prepare us for the eventual live demo per each API page.

Reply:
Done!